lib/core/imports/index.js

```diff
@@ -1,4 +1,14 @@
-/* global axe */
+/**
+ * Namespace `axe.imports` which holds required external dependencies
+ *
+ * @namespace imports
+ * @memberof axe
+ */
+export { default as axios } from 'axios';
```

Comment:
This is my favorite part of this PR. Being able to `import` 3rd party tools will hugely improve our workflows. :heart:

Reply:
I know! Webpack is pretty awesome at that.